combat-trainer.lic

```diff
@@ -20,6 +20,10 @@ class SetupProcess
     @cycle_armors = settings.cycle_armors
     echo("  @cycle_armors: #{@cycle_armors}") if $debug_mode_ct
     @last_cycle_time = Time.now - 125
+    
+    @warhorn = settings.warhorn
```

Comment:
Can you update `base.yaml` with a default value for this setting?

Reply:
based on @jayRyan24 feedback, we will be using a null value as a false check for using a warhorn. Im a Ruby noob though, so I can add it if thats not going to cause issues.

Reply:
Still should be there in base. It needs to just be `warhorn:` in the base yaml. This let's people know the different settings.

Reply:
done!